app/src/main/webapp/WEB-INF/jsp/tiles/local-masthead.jsp

```diff
@@ -40,7 +40,7 @@
                     <li id="local-nav-experiments"><a href="${pageContext.request.contextPath}/experiments"><i class="icon icon-functional padding-right-medium" data-icon="C"></i>Browse experiments</a></li>
                     <li id="local-nav-download"><a href="${pageContext.request.contextPath}/download"><i class="icon icon-common padding-right-medium" data-icon="="></i>Download</a></li>
                     <li id="local-nav-release-notes"><a href="${pageContext.request.contextPath}/release-notes.html"><i class="icon icon-generic padding-right-medium" data-icon=";"></i>Release notes</a></li>
-                    <li id="local-nav-help"><a href="${pageContext.request.contextPath}/help.html"><i class="icon icon-generic padding-right-medium" data-icon="?"></i>Help</a></li>
+                    <li id="local-nav-help"><a href="${pageContext.request.contextPath}/help.html?section=${section}"><i class="icon icon-generic padding-right-medium" data-icon="?"></i>Help</a></li>
```

Comment:
Yes, sure, because we would like to point to the specific section in the help page. Can you tell the difference when you run the server with and without `section=external-data`?